src/classes/Users.ts

```diff
@@ -152,7 +152,8 @@ export default class Users extends Base {
    * @returns A user
    */
   public async getUser(user: number | string, options?: GetUserOptions): Promise<UserExtended> {
-    user = z.number().parse(user);
+    if(typeof user === "number") user = z.number().parse(user);
+    else user = z.string().parse(user);
```

Comment:
Instead of using if/else, you can also use a union schema.

```ts
user = z.union([z.string(), z.number()]).parse(user);
```

Reply:
I am sorry, I am not very familiar with Zod and I did it. I also added JSDoc for the options parameter for intellisense